h_periodic/h_beat.py

```diff
@@ -1,9 +1,18 @@
 """Celery beat scheduler process configuration."""
 
+import sys
 from datetime import timedelta
+from os import environ
 
 from celery import Celery
 
+# We don't want periodic tasks for h-qa because h-qa and h-prod share the same
+# database and search index.
+# See https://github.com/hypothesis/playbook/issues/436
+# So we'll set this DISABLE_H_BEAT envvar on h-qa.
+if "DISABLE_H_BEAT" in environ:
```

Comment:
Might be nice to blart something out here in case someone ends up scratching their head as to why this isn't starting.